spack/ci-spack-pyprecice-deps-1804.dockerfile

```diff
@@ -1,11 +1,11 @@
 # Build stage with Spack pre-installed and ready to be used
-FROM spack/ubuntu-bionic:latest 
+FROM spack/ubuntu-bionic:latest
 
 # Mount the current sources into the build container
 # and build the default environment
 ADD ./spack/repo /py-pyprecice-repo
 RUN spack --color=always env create --without-view ci && \
-    spack --color=always -e ci add py-pyprecice@develop target=x86_64 && \
     spack --color=always -e ci repo add /py-pyprecice-repo && \
```

Comment:
I think it still has to be there such that `spack --color=always -e ci install --fail-fast --only=dependencies` can resolve the dependencies of `py-pyprecice`.  If we do not add the package and the repository to the currecnt environment `ci` Spack will use the wrong package recipe and it will not build anything as the environment `ci` would be empty.